src/engine.rs

```diff
@@ -319,61 +319,91 @@ impl Engine {
 }
 
 impl evm::backend::Backend for Engine {
+    /// Returns the gas price.
+    ///
+    /// This is currently zero, but may be changed in the future. This is mainly
+    /// because there already is another cost for transactions.
     fn gas_price(&self) -> U256 {
         U256::zero()
     }
 
+    /// Returns the origin address that created the contract.
     fn origin(&self) -> Address {
         self.origin
     }
 
+    /// Returns a block hash from a given index.
+    ///
+    /// Currently this returns zero, but may be changed in the future.
     fn block_hash(&self, _number: U256) -> H256 {
         H256::zero() // TODO: https://github.com/near/nearcore/issues/3456
     }
 
+    /// Returns the current block index number.
     fn block_number(&self) -> U256 {
         U256::from(sdk::block_index())
     }
 
+    /// Returns a mocked coinbase which is the first 160 bits of
+    /// `keccak256(b"aurora")`.
+    ///
+    /// It is not possible to return the address of the current block's miner in
+    /// NEAR as it isn't relevant.
     fn block_coinbase(&self) -> Address {
-        Address::zero()
+        Address([
+            0x2b, 0x0b, 0xf3, 0xb8, 0xff, 0xaa, 0x4f, 0x3d, 0x1f, 0x97, 0x76, 0x0d, 0x44, 0x44,
+            0x58, 0x84, 0x43, 0xc3, 0xa9, 0x12,
+        ])
```

Comment:
This is incorrect. The EVM address for the `aurora` account is _0x4444588443C3a91288c5002483449Aba1054192b_.

The EVM address derivation does not use the initial bytes. Relevant references:
- https://doc.aurora.dev/develop/compat/evm#coinbase
- https://github.com/aurora-is-near/aurora.js/blob/7f7bc70/src/account.ts#L24